api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -5540,11 +5540,21 @@
             ],
             "description": "HTTPGet specifies the http request to perform."
           },
+          "initialDelayMilliseconds": {
+            "description": "Length of time (in milliseconds) before health checking is activated.",
+            "format": "int32",
+            "type": "integer"
+          },
           "initialDelaySeconds": {
             "description": "Number of seconds after the container has started before liveness probes are initiated. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes",
             "format": "int32",
             "type": "integer"
           },
+          "periodMilliseconds": {
```

Comment:
`periodMilliseconds` is kind of a bad name since it augments `periodSeconds`. Can we change the naming so that it is more clear this is the case? something like `additionalMilliseconds` or `augmentedMilliseconds`.

Reply:
I've been holding off making updates here until the KEP merges. Naming is something we've gone back and forth on a lot, so I'd weigh in over there too: https://github.com/kubernetes/enhancements/pull/3067.

As of now, the general consensus eventually seems to have settled on `periodOffsetMilliseconds`.

Reply:
That sounds reasonable to me.